e2e_test.go

```diff
@@ -372,3 +372,34 @@ func (suite *e2eSuite) TestResponseAutoChunking() {
 	suite.Assert().Equal(http.StatusOK, rsp.StatusCode)
 	suite.Assert().Contains(rsp.TransferEncoding, "chunked")
 }
+
+func BenchmarkRequestResponse(b *testing.B) {
+	b.ReportAllocs()
+	svc := Service(func(req Request) Response {
+		rsp := req.Response(nil)
+		rsp.Header.Set("a", "b")
+		rsp.Header.Set("b", "b")
+		rsp.Header.Set("c", "b")
+		return rsp
+	})
+	addr := &net.UnixAddr{
+		Net:  "unix",
+		Name: "/tmp/typhon-test.sock"}
+	l, _ := net.ListenUnix("unix", addr)
```

Comment:
@obeattie TIL you can listen on a unix socket instead of network address and the rest of the library works the same! 💡 Will there be any difference in benchmarks because of the change in socket type? Is there a case in which this benchmark reports a better number while the actual RTT gets worse? Just curious.